vendor/github.com/AviatrixSystems/go-aviatrix/goaviatrix/version.go

```
@@ -44,6 +44,8 @@ type AviatrixVersion struct {
 func (c *Client) Upgrade(version *Version) error {
 	path := ""
 	if version.Version == "" {
+		return nil
```

I think you should throw an error here